src/Node.php

```diff
@@ -354,7 +354,7 @@ public function jsonSerialize() {
     }
 
     /**
-     * Get the end index of a Node.
+     * Get the end position of a Node.
```

Comment:
This method uses the term `Position`. Maybe  we should change the API to `get[Start|End]Offset`?